src/charts/tableView.js

```diff
@@ -12,14 +12,16 @@ define(['jquery', 'moment', 'lokijs', 'charts/chartingRequestMap', 'websockets/s
     table.animate({left: '0'}, 250);
     chart.animate({left: '-100%'}, 250);
     refresh_table(dialog, key); /* clear table and fill it again */
+    dialog.parent().css('width', 520);
```

Comment:
Please only increase the width of dialog when the table width is bigger than the dialog.
And the content of table does not fit.
Also you may consider using a jquery animation :)